spec/ios_merge_strings_files_spec.rb

```diff
@@ -1,41 +1,39 @@
-describe Fastlane do
-  describe Fastlane::FastFile do
-    let(:test_data_dir) { File.join(File.dirname(__FILE__), 'test-data', 'translations', 'ios_l10n_helper') }
+describe Fastlane::Actions::IosMergeStringsFilesAction do
```

Comment:
With the new `run_described_action` helper, we get to use the actual type we're testing here which I think makes the test a tiny bit clearer.